resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/kubernetes/Client.scala

```diff
@@ -184,7 +188,7 @@ private[spark] class Client(
           kubernetesClient.pods().createNew()
             .withNewMetadata()
               .withName(kubernetesAppId)
-              .withLabels(selectors)
+              .addToLabels(resolvedSelectors)
```

Comment:
In this context it's fine but this change is accidental and probably should be reverted to reduce the unnecessary diff.